fe/fe-core/src/main/java/com/starrocks/common/proc/BalanceStatProcNode.java

```diff
@@ -0,0 +1,39 @@
+// Copyright 2021-present StarRocks, Inc. All rights reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package com.starrocks.common.proc;
+
+import com.google.common.collect.ImmutableList;
+import com.starrocks.clone.TabletScheduler;
+import com.starrocks.common.AnalysisException;
+
+public class BalanceStatProcNode implements ProcNodeInterface {
+    public static final ImmutableList<String> TITLE_NAMES = new ImmutableList.Builder<String>()
+            .add("StorageMedium").add("BalanceType").add("Balanced").add("PendingTablets").add("RunningTablets").build();
```

Comment:
Can we add another attribute (like "description") to describe what each BalanceType means?

Reply:
Make balance type more clear.